easybuild/easyblocks/g/gromacs.py

```diff
@@ -139,10 +141,14 @@ def sanity_check_step(self):
         libnames = ['gromacs']
         if LooseVersion(self.version) < LooseVersion('5.0'):
             libnames = ['gmxana', 'gmx', 'gmxpreprocess', 'md']
+
+        libprefix = ''
+        if LooseVersion(self.version) >= LooseVersion('5.0'):
+            libprefix = get_platform_name()
         libs = ['lib%s%s.a' % (libname, suff) for libname in libnames]
         custom_paths = {
             'files': ['bin/%s%s' % (binary, suff) for binary in ['editconf', 'g_lie', 'genbox', 'genconf', 'mdrun']] +
```

Comment:
feedback provided by @rapostolo, he had more suggestions for improvements, more info later...